File: mobile/lib/database/flight.dart
class Flight {
  final int? id;
  final int? idRute;
  final String airplaneLicense;
  final String gate;
  final String schedule;
  final String status;
  final int deals;

  Flight(
      {required this.id,
      required this.idRute,
      required this.airplaneLicense,
      required this.gate,
      required this.schedule,
      required this.status,
      required this.deals});

  factory Flight.fromMap(Map<String, dynamic> json) => Flight(
      id: json['id'],
      idRute: json['idRute'],
      airplaneLicense: json['airplaneLicense'],
      gate: json['gate'],
      schedule: json['schedule'],
      status: json['status'],
      deals: json['deals']);

  Map<String, dynamic> toMap() {
    return {
      'id': id,
      'idRute': idRute,
      'airplaneLicense': airplaneLicense,
      'gate': gate,
      'schedule': schedule,
      'status': status,
      'deals': deals,
    };
  }
}

File: api/Entities/Flight.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace api.Entities
{
    // Flight structure
    public partial class Flight
    {
        public Flight()
        {
            Books = new HashSet<Book>();
        }
        // Unique Flight ID
        public int Id { get; set; }
        // Assigned Airplane License
        public string AirplaneLicense { get; set; }
        // ID of the Assigned Rute
        public int IdRute { get; set; }
        // Boarding Gate
        public string Gate { get; set; }
        // Departure time
        public DateTime Schedule { get; set; }
        // Flight Status
        public string Status { get; set; }
        // Flight Deal
        public int? Deals { get; set; }

        public virtual Plane AirplaneLicenseNavigation { get; set; }
        public virtual Rute IdRuteNavigation { get; set; }
        public virtual ICollection<Book> Books { get; set; }
    }
}
